pkgs/applications/misc/conduktor/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchzip, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
```

```suggestion
{ stdenv, lib, fetchzip, fetchurl, jdk11, unzip, makeWrapper, makeDesktopItem, copyDesktopItems }:
```

this fixes the current eval error.

---

fixed, thanks